src/main/java/dinkplugin/DinkPlugin.java

```diff
@@ -76,6 +84,37 @@ DinkPluginConfig provideConfig(ConfigManager configManager) {
         return configManager.getConfig(DinkPluginConfig.class);
     }
 
+    @Subscribe
+    public void onConfigChanged(ConfigChanged event) {
+        if (!"dinkplugin".equals(event.getGroup()))
+            return;
+
+        String key = event.getKey();
+        String value = event.getNewValue();
+        GameState gameState = client.getGameState();
+
+        if ("combatTaskEnabled".equals(key) && "true".equals(value) && gameState == GameState.LOGGED_IN) {
+            clientThread.invokeLater(() -> {
+                if (client.getVarbitValue(Varbits.COMBAT_ACHIEVEMENTS_POPUP) < 1) {
+                    log.warn("Combat Achievements popup is not enabled in RuneScape settings; Dink Combat Task notifier will not fire.");
+                }
```

Comment:
`COMBAT_ACHIEVEMENT_POPUP` is not synonymous with combat achievements as a whole being enabled, from what I can see there's no way to disable them.

Collection Log has a popup setting, but it is separate from the write-to-chat setting.